packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -366,6 +372,19 @@ export class Dialog
   // #endregion
 
   // #region Private Methods
+
+  private handleFocusTrapDisabled(focusTrapDisabled: boolean): void {
```

Comment:
no need to pass focusTrapDisabled value, just use `this.focusTrapDisabled` within the function.